client/src/i18n/en-us/index.js

```diff
@@ -207,4 +208,33 @@ export default {
   },
   general_failure: "Oops, there was an error.",
   loading: "Loading...",
+  /*********************
+   * Notifications follow the format of:
+   * <type>: {
+   *    short: "",
+   *    long: "",
+   * }
+   */
+  notifications: {
+    review: {
+      requested: {
+        short: "{user.username} has requested your review on {object.name}",
+      },
+      commentReply: {
+        short: "{user.username} has replied to your comment on {object.name}",
+      },
+    },
+    submission: {
+      new: {
+        short:
+          "{user.username} has submitted {object.name} to {object.publication.name}",
+      },
+      approved: {
+        short: "{object.name} has been approved",
+      },
+      resubmission: {
```

Comment:
We should be mindful of and avoid mixing past verbs (`approved`) and nouns (`resubmission`) in our APIs.

Choose either nouns: 

```
request
reply
creation
approval
resubmission
```

or past verbs:

```
requested
replied
created
approved
resubmitted
```

If it were up to me, I would choose nouns because some languages don't have past tense.

Reply:
Since they're keys and not part of the translation I'm going to lean towards past verb on this bc in the notification sense, the action has happened. I'll push an update.